corehq/apps/linked_domain/applications.py

```diff
@@ -12,11 +12,14 @@
 )
 
 
-def get_master_app_briefs(domain_link):
+def get_master_app_briefs(domain_link, family_id):
     if domain_link.is_remote:
         apps = get_brief_apps(domain_link.master_domain, domain_link.remote_details)
     else:
         apps = get_brief_apps_in_domain(domain_link.master_domain, include_remote=False)
+
+    apps = [app for app in apps if family_id in [app._id, app.family_id]]
```

Comment:
could combine this with the filtering below

Reply:
31c30ed3d65